internal/command/cmd_test.go

```diff
@@ -0,0 +1,20 @@
+package command
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestCmd will test a command retry and cancellation
+func TestCmd(t *testing.T) {
+	cmd := Cmd{"sleep 2 && ls", "", nil, nil}
+	err := cmd.RetryCommand(1, 1)
+	require.Error(t, err)
```

Comment:
we expect an error here because the command times out before it could be retried? adding a couple of comments here could make it clearer.

Reply:
>we expect an error here because the command times out before it could be retried

Yes, an error occurs because the command is cancelled before it ends the execution, for every retry intentionally.